ProteGO/UI/History/SendHistoryResultDialog/ViewController/SendHistoryResultDialogViewController.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+import RxSwift
+
+final class SendHistoryResultDialogViewController: UIViewController, CustomView {
+
+    typealias ViewClass = SendHistoryResultDialogView
+
+    private let isSuccessAlert: Bool
```

Comment:
Tak tylko przeglądam sobie ten kod. Wrzucenie tego stanu w ViewModel i zbindowanie to byłby overkill? 🙂

Reply:
chyba tak:)

Wydaje mi sie ze bindowanie byłoby uzasadnione gdyby ten alert miał prawo się zmienić po jego pokazaniu.

A ten dialog się nie zmienia, bo to jest status, który otrzymaliśmy po wysyłaniu historii na serwer. Jak przyszedł nam np. błąd to nie zamieni się on juz na sukces ;)